src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/JCloudsArtifactManagerTest.java

```diff
@@ -24,8 +24,6 @@
 
 package io.jenkins.plugins.artifact_manager_jclouds.s3;
```

Comment:
another improvement could be to extend `AbstractIntegrationTest` to get rid of some duplication but maybe another PR.

Reply:
> extend `AbstractIntegrationTest`

Not likely; that is based on `RealJenkinsRule`, this is not.